omid/Dockerfile

```diff
@@ -0,0 +1,85 @@
+# syntax=docker/dockerfile:1
+FROM stackable/image/java-base AS builder
+
+ARG PRODUCT
+
+RUN microdnf update && \
+    microdnf install tar maven java-11-openjdk-devel && \
+    microdnf clean all
+
+USER stackable
+WORKDIR /stackable
+
+SHELL ["bash", "-euo", "pipefail", "-c"]
+
+RUN curl -L https://dist.apache.org/repos/dist/release/phoenix/phoenix-omid-${PRODUCT}/phoenix-omid-${PRODUCT}-src.tar.gz | tar -xzC .
+
+WORKDIR /stackable/phoenix-omid-${PRODUCT}
+RUN mvn package -Phbase-2 -DskipTests && \
+    tar -xf tso-server/target/omid-tso-server-${PRODUCT}-bin.tar.gz -C /stackable
+
+FROM stackable/image/java-base
+
+ARG PRODUCT
+ARG RELEASE
+
+LABEL name="Apache Omid" \
+      maintainer="info@stackable.tech" \
+      vendor="Stackable GmbH" \
+      version="${PRODUCT}" \
+      release="${RELEASE}" \
+      summary="The Stackable image for Apache Omid." \
+      description="This image is deployed by the Stackable Operator for Apache HBase."
+
+# https://github.com/hadolint/hadolint/wiki/DL4006
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+RUN microdnf update && \
+    microdnf install tar gzip zip && \
+    microdnf clean all
+
+RUN ln -s /usr/bin/python3 /usr/bin/python && \
+    ln -s /usr/bin/pip3 /usr/bin/pip
+
+ENV HOME=/stackable
+
+COPY --chown=stackable:stackable omid/stackable /stackable
+COPY omid/licenses /licenses
+
+USER stackable
+WORKDIR /stackable
+
+RUN ln -s /stackable/omid-tso-server-${PRODUCT} /stackable/omid-tso-server && \
+    curl https://repo.stackable.tech/repository/packages/jmx-exporter/jmx_prometheus_javaagent-0.16.1.jar \
```

Comment:
I think this is a variable with the other images

Reply:
I don't think it was at the time, but worth factoring it out.

Reply:
2bad8bffec125d59aa8ea9ceacaa3f866b1bb95b